indicator/src/main/java/com/fuzz/indicator/OffsetEvent.java

```diff
@@ -0,0 +1,11 @@
+package com.fuzz.indicator;
+
+/**
+ * This represents a single action or {@link android.view.MotionEvent} or whatever
+ * that would trigger a call to
+ * {@link CutoutViewIndicator#showOffsetIndicator(int, float)}.
+ *
+ * @author Philip Cohn-Cort (Fuzz)
+ */
+public interface OffsetEvent {
```

Comment:
Right now this is just a marker interface, but it doesn't have to be. Maybe pull up the `getPosition` method?